perf_test/sparse/KokkosSparse_run_spgemm.hpp

```diff
@@ -319,8 +319,8 @@ crsMat_t3 run_experiment(
 	  row_mapC = lno_view_t
 			  ("non_const_lnow_row",
 					  m + 1);
-	  entriesC = lno_nnz_view_t ("");
-	  valuesC = scalar_view_t ("");
+	  entriesC = lno_nnz_view_t ();
```

Comment:
Simple request, can we name it instead of removing it ? Will help in debugging later.

Reply:
Sure, I just added back the labels and allocated them to size 0. After this change I rebuilt everything and ran the SPGEMM driver on my local machine (CUDA/OpenMP), with and without deprecated.